app/reducers/channels.js

```diff
@@ -191,7 +192,18 @@ export const openChannel = ({ pubkey, host, local_amt }) => (dispatch, getState)
   dispatch(openingChannel())
   dispatch(addLoadingPubkey(pubkey))
 
-  ipcRenderer.send('lnd', { msg: 'connectAndOpen', data: { pubkey, host, localamt } })
+  // Grab the activeConnection type from our local store.
+  // If the active connection type is local (light clients using neutrino)
+  // we will flag manually created channels as private. Other connections
+  // like remote node and BTCPay Server we will announce to the network as
+  // these users are using Zap to drive nodes that are online 24/7
+  const store = new Store({ name: 'settings' })
+  const { type } = store.get('activeConnection', null)
```

Comment:
If `activeConnection` is not set in the store, `store.get` should return an empty object, instead of `null`
 
```javascript
const { type } = store.get('activeConnection',{})
```

Otherwise, we would be trying to destructure `null` which would cause an error.